gulpfile.js

```diff
@@ -403,7 +403,8 @@ function thirdPartyBootstrap(watch, shouldMinify) {
   console.log('Processing ' + input);
   var html = fs.readFileSync(input, 'utf8');
   var min = html;
-  min = min.replace(/\.\/integration\.js/g, './f.js');
+  min = min.replace(/\.\/integration\.js/g,
```

Comment:
I had to update this a bit for testing and added lots of comments.

This change will need to be tested in op-in prod. Should be a no-op, but changes the contents of the file. Previously we always used a relative URL.